File: Algorithm/Baekjoon/Python/입출력과 사칙연산/나머지.py
i = input().split()
A = int(i[0])
B = int(i[1])
C = int(i[2])

print((A + B) % C)
print(((A % C) + (B % C)) % C)
print((A * B) % C)
print(((A % C) * (B % C)) % C)

File: Algorithm/Baekjoon/Java/입출력과 사칙연산/나머지.java
import java.util.*;

public class 나머지 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        String[] str = sc.nextLine().split(" ");
        int A = Integer.parseInt(str[0]);
        int B = Integer.parseInt(str[1]);
        int C = Integer.parseInt(str[2]);

        System.out.println((A + B) % C);
        System.out.println(((A % C) + (B % C)) % C);
        System.out.println((A * B) % C);
        System.out.println(((A % C) * (B % C)) % C);
        sc.close();
    }
}
